common/src/main/java/org/apache/uniffle/common/ServerStatus.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.uniffle.common;
+
+public enum ServerStatus {
+  NORMAL_STATUS,
+  DECOMMISSIONING;
```

Comment:
> For the state that all the app has already been finished, but the shuffle server is configured to not exit itself?

In what case it don't need to exit.

Reply:
> is it possible to add a DECOMMISSIONED status here?
For the state that all the app has already been finished, but the shuffle server is configured to not exit itself?

+1. Because maybe in the future, we will have a dashboard to show those decommissioned server. This could indicate the former state of servers.

Reply:
How about change `NORMAL_STATUS` to `RUNNING` ?

Reply:
> How about change `NORMAL_STATUS` to `RUNNING` ?

I have considered it before, but i think it is inappropriate.